vuepress/docs/.vuepress/theme/layouts/IndexPost.vue

```diff
@@ -0,0 +1,37 @@
+<!-- Override and extend the default theme Layout -->
```

Comment:
Not sure where to put this comment but it might be good to have a title to the page, something like Entando Tech Blog Series or something that shows that they landed on the right page